package.json

```diff
@@ -30,17 +30,12 @@
     "headless.js"
   ],
   "scripts": {
-    "start-webpack": "(cd examples/main && yarn && npm run start-local)",
-    "start": "npm run build-watch && budo dist/bundle.js:dist/lumagl.js --live --open --port 3000 --watch-glob '**/*.{html,css,js,glsl}' ",
-    "build-watch": "babel src -d dist --source-maps inline --copy-files --watch & babel examples -d dist --source-maps inline --copy-files --watch &",
+    "start": "cat < docs/get-started/examples.md",
```

Comment:
> My understanding is that the layer browser also exists to ensure compatibility between layers, so if we remove it as an example entry point we'll still need to keep it around for that use.

Good point. Wasn't planning on removing any examples (least of all the layer browser). Just changing the top level `npm start` command from running the layer browser to showing the docs to avoid the technical complications with dual installs.

This seems more like a process issue, e.g.:
* At the moment we mainly test that all the examples work before we release but since an increasing amount of users are working on master it is becoming important to keep it in good condition.
* We could publish a test guideline for contributions and ask contributors do a manual test run using the layer browser on any PRs they submit. 
* Or update our internal guidelines to require that we run the layer browser before merging non-trivial PRs.